src/Client/Components/Layout/SiteFooter.razor

```diff
@@ -85,24 +85,22 @@
             <div class="col" id="footer-learn-col">
                 <div class="pb-2 font-weight-bold">Learn</div>
                 <div><Href href="https://start.steeltoe.io/" NewWindow="true">Steeltoe Initializr</Href></div>
-                <div><Href href="@(docsSite.GuidesHome)">Guides</Href></div>
-                <div><Href href="@(docsSite.DocsHome)">Documentation</Href></div>
-                <div><Href href="@(docsSite.ApiBrowserHome)">API Browser</Href></div>
-                <div><Href href="@(docsSite.BlogHome)">Blog</Href></div>
+                <div><Href href="@(_docsSite.GuidesHome)">Guides</Href></div>
+                <div><Href href="@(_docsSite.DocsHome)">Documentation</Href></div>
+                <div><Href href="@(_docsSite.ApiBrowserHome)">API Browser</Href></div>
+                <div><Href href="@(_docsSite.BlogHome)">Blog</Href></div>
             </div>
             <div class="col" id="footer-projects-col">
                 <div class="pb-2 font-weight-bold">Projects</div>
                 <div><Href href="/app-configuration">Application Configuration</Href></div>
                 <div><Href href="/circuit-breakers">Circuit Breakers</Href></div>
-                @*<div><Href href="/cloud-management">Distributed Tracing</Href></div>*@
-                <div><Href href="/logging">Dynamic Logging</Href></div>
                 <div><Href href="/cloud-management">Management</Href></div>
                 <div><Href href="/messaging">Messaging</Href></div>
-                <div><Href href="/fileshares">Network File Sharing</Href></div>
+                <div><Href href="@(_docsSite.BaseAddress + "/api/v3/fileshares/")">Network File Sharing</Href></div>
```

Comment:
I'm seeing this path occur in multiple files. Would it make sense to extract it into a property, similar to `DocsStreamHome`?

Reply:
It's used twice, but ok